hokusai/cli/staging.py

```diff
@@ -34,7 +35,7 @@ def delete(filename, verbose):
 
 
 @staging.command(context_settings=CONTEXT_SETTINGS)
-@click.option('--check-branch', type=click.STRING, default="master", help='Check branch before updating (default: master)')
+@click.option('--check-branch', type=click.STRING, default=detect_branch(), help='Check branch before updating (default: main)')
```

Comment:
Minor: the `default: main` help text isn't quite accurate. Maybe it should say "defaults to the origin remote's default branch" now.

Reply:
You're right, I updated it 👍